src/data/stackexchange/stack_exchange_parse.py

```diff
@@ -278,30 +224,36 @@ def main(args):
 
     if args.skipparse == False:
```

Comment:
I believe your new logic of verifying completed parsing components removes the need to have a skipparse command line argument